security/comms/typha-node-tls.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Provide TLS certificates for Typha and Node
+description: Add TLS certificates to secure communications between if you are using Typha to scale your deployment.
```

Comment:
Maybe this should be `Add TLS certificates to secure communications between Typha and Node.`